### DISCUSSION TITLE
Security the cybercrime

### DISCUSSION THREAD

**Discussion by ReginaIverson** [community/community#63525] [Code Security]
### Select Topic Area

Show & Tell

### Body

Someone is taking into my account and scanning my apps so if you could please close it or look into it for me #

**Comment by ReginaIverson** [comment]
I need to know what to do if someone is hacking my accounts and scanning my apps

**Comment by elderguardian** [comment]
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account
